### PR TITLE
chore(l10n-zh): update meta docs

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

We have invited new member to our team, it's time to update the meta contents.

### Related issues and pull requests

- #18047
- mdn/content#32530

### CODE REVIEW

File: files/zh-cn/mdn/community/contributing/translated_content/index.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ slug: MDN/Community/Contributing/Translated_content
 
 ### 汉语（zh-CN、zh-TW）
 
-- 讨论组：[Telegram (MozTW L10n channel)](https://moztw.org/community/telegram/)、[Telegram (Mozilla China channel)](https://t.me/mozilla_china)
-- 目前的志愿者：[Irvin](https://github.com/irvin)、[t7yang](https://github.com/t7yang)、[yin1999](https://github.com/yin1999)、[jasonren0403](https://github.com/jasonren0403)
+- 讨论组：[Telegram (`Mozilla China`)](https://t.me/mozilla_china)、[Telegram (`MozTW L10n`)](https://moztw.org/community/telegram/)
```
Comment (yin1999) [inline files/zh-cn/mdn/community/contributing/translated_content/index.md:27]:
```suggestion
- 讨论组：[Telegram（`Mozilla China`）](https://t.me/mozilla_china)、[Telegram（`MozTW L10n`）](https://moztw.org/community/telegram/)
```

**Review by yin1999 — COMMENTED** [review]
Let's also fix this:

**Review by yin1999 — APPROVED** [review]
LGTM for `l10n-zh`!

**Review by saionaro — APPROVED** [review]
LGTM for ru

**Review by tristantheb — APPROVED** [review]
lgtm for fr

**Review by bsmth — APPROVED** [review]
LGTM, thank you!

**Review by JuanVqz — COMMENTED** [review]
@Graywolf9

**Review by JuanVqz — APPROVED** [review]
Looks good for ES members

**Comment by jasonren0403** [comment]
waiting for approvment from @mdn/yari-content-ja 🚶

**Review by mfuji09 — APPROVED** [review]
@jasonren0403 
Sorry too late replay.
Looks good for me!

**Comment by bsmth** [comment]
Thanks, everyone 👏🏻

**Pull request merged**